.github/workflows/ci.yml

```
@@ -13,30 +13,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', 'pypy-3.9']
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.9']
```

```suggestion
        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.10']
```
https://pypy.org/download.html